static/app/views/settings/components/settingsNavigationGroup.tsx

```diff
@@ -1,3 +1,5 @@
+import {useParams} from 'react-router-dom';
```

Comment:
looks like you're using `import {useParams} from 'sentry/utils/useParams';` and `import {useParams} from 'react-router-dom';` in this pr

Reply:
You're right! fixed